examples/bri-3/prisma/schema.prisma

```diff
@@ -18,6 +18,11 @@ model BpiSubject {
   publicKey   String
 }
 
+model BpiAccount {
```

Comment:
@skosito 
- The model structure is defined in [R251](https://github.com/eea-oasis/baseline-standard/blob/main/core/baseline-core-v1.0-psd01.md#r251) ... the model (and migration above) does not comply with that. The value account balance(s) is actually incorrect because of D35.  
- Also, if we do not define the relationship between `BPIAccount` and its state here, how do we intend to achieve that? IMHO, just put a pointer object to the state object here e.g. could be a smart contract on Ethereum or on the BPIs zkEVM
- Also, we do not have a place for the required vector commitment scheme for the BPI Account state? In other words, where do we intend to store it how do we construct it? IMHO, the easiest way is to represent it is through a merkle tree where each BPI Account is a leaf, that kills all bords with one so to speak

Based on your suggestions the handlers etc below change too

cc @ognjenkurtic @Kasshern @Manik-Jain @Ybittan

Reply:
@Therecanbeonlyone1969 this is based on already approved model  https://github.com/eea-oasis/baseline/pull/507 and discussion here https://docs.google.com/document/d/1DCi0Ap9EK2cBxxEVzmOPkdkcl1hnv7-qoELGHODb1PM/edit where the model for first version is clearly defined and approved

if we follow the process i suggest we merge current work and open issue for additional changes - there are 3 PRs waiting to be merged related to accounts and bpi subject accounts with relations between them defined which are correct, if i understand your comments i need to add more fields to model and db, which can be follow up work and will result in additional migrations

Reply:
on second look, a lot of these fields to add would be placeholders anyways, because we dont have adequate db models to connect them, so we can just add them as we go

Reply:
@Therecanbeonlyone1969 yes, let's please stick with the proposed models from the document @skosito outlined. As soon as CRUD milestone is reached we will start tackling all the intricate details of state storage and other MUST requirements outlined in the standard. These will be tackled in separate smaller issues to avoid tackling mulitple complex topics at the same time and to enable detailed discussions in the issues before implementation even begins. Dont worry,. no MUST requirement outlined in the scope document  will be left behind ;).

Reply:
Ok. That is fine. It just needs to be tracked